vembrane/__init__.py

```diff
@@ -84,6 +86,30 @@ def filter_vcf(vcf: VariantFile, expression: str) -> Iterator[VariantRecord]:
         yield record
 
 
+def statistics(vcf, records, n):
+    annotation_keys = get_annotation_keys(vcf.header)
+    if n == 0:
+        print(*annotation_keys, sep="\n")
+        return
+
+    counter = defaultdict(lambda: defaultdict(lambda: 0))
+    for i, record in enumerate(records):
+        if i >= n: # process only the first n records
+            break
+        for annotation in record.info["ANN"]:
+            for key, value in zip(annotation_keys, parse_annotation_entry(annotation)):
+                if value:
+                    counter[key][value] += 1
+    
+    for key, subdict in counter.items():
+        print(key)
+        if len(subdict) <= 10:
```

Comment:
Is 10 a good threshold?
Apart from that, I prefer introducing constants (somewhere), makes it easier to change thresholds such as this one later on.

Reply:
I prefer to make this adjustable by the user :)

Reply:
I will change the branch, so that it will write an additional statistic file.

Reply:
> I prefer to make this adjustable by the user :)

as long as the cli doesn't have too many options / isn't cluttered ;)

Reply:
let me show you the snakemake --help
:D